images/linux/toolsets/toolset-2004.json

```diff
@@ -85,10 +85,13 @@
         "additional_tools": [
             "cmake;3.10.2.4988404",
             "patcher;v4",
-            "ndk;21.3.6528147",
             "platform-tools",
             "cmdline-tools;latest"
-        ]
+        ],
+        "ndk": {
+            "lts": "21.3.6528147",
+            "latest": "22.0.7026061"
```

Comment:
Can we specify only the major version here and install the latest minor for it? Because lts and latest can be updated through the year.